tests/integrations/test_crossed_integrations.py

```diff
@@ -63,13 +80,33 @@ def get_topic(span) -> str | None:
     def setup_produce(self):
         """
         send request A to weblog : this request will produce a kafka message
-        send request B to python buddy, this request will consume kafka message
+        send request B to library buddy, this request will consume kafka message
         """
-
-        python_buddy = _PythonBuddy()
-
-        self.production_response = weblog.get("/kafka/produce", params={"topic": self.WEBLOG_TO_BUDDY_TOPIC})
-        self.consume_response = python_buddy.get("/kafka/consume", params={"topic": self.WEBLOG_TO_BUDDY_TOPIC})
+        timeout = time.time() + 300
+
+        self.production_response = None
+        self.consume_response = None
+        while (
```

Comment:
This loop bother me. Which buddy does not have a good startup sequence ?

Reply:
The looping has been removed, everything is working properly now without the looping